src/pages/help-centre/index.tsx

```diff
@@ -260,7 +262,7 @@ const HelpCentre = () => {
             <Desktop breakpoint={'tabletS'}>
                 <Community />
             </Desktop>
-            <DidntFindYourAnswerBanner />
+            {!isDerivGo && <DidntFindYourAnswerBanner />}
```

Comment:
```suggestion
            {isDerivGo && <DidntFindYourAnswerBanner />}
```
No need to coerce if you can guarantee that it is a boolean

Reply:
As discussed in person, We need to hide this section only for DerivGO users so we need to reverse the condition here🙏